Amazon.IonObjectMapper/Attribute/IonIgnoreAttribute.cs

```diff
@@ -15,6 +15,10 @@ namespace Amazon.IonObjectMapper
 {
     using System;
 
+    /// <summary>
+    /// Attribute to identify a .NET property that should be ignored
+    /// by the serializer.
```

Comment:
During serialization and deserialization?

Reply:
Done.